osis-core/src/main/java/com/vmware/osis/scality/AppEnv.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache License 2.0
  */
 
-package com.vmware.osis.platform;
+package com.vmware.osis.scality;
```

Comment:
Who are we to define a `com.vmware.osis.scality` package whilst we don't own the `(osis.)vmware.com` domain?

Reply:
@NicolasT 
Thank you. Updated all packages from `com.vmware.*` to `com.scality.*`